src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+
+/**
+ * Dook the all purpose chatbot to serve your human needs. Made from the finest IDEs, Dook will be the most
+ * efficient task list you have ever laid your hands upon.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * To instantiate a Dook instance with a given memory file path.
+     *
+     * @param filePath  File path of memory file.
+     */
+    public Duke(String filePath) {
```

Comment:
Maybe you can consider changing the class name to Dook as well to match the name that you have given it? I think it's better to match the class name with what you actually want to call him.